torch/distributed/distributed_c10d.py

```diff
@@ -2219,14 +2219,23 @@ def all_gather(tensor_list, tensor, group=None, async_op=False):
         work.wait()
 
 
-def _all_gather_base(output_tensor, input_tensor, group=None, async_op=False):
+def all_gather_into_tensor(output_tensor, input_tensor, group=None, async_op=False):
```

Comment:
let's update the PyTorch web documentation by adding this to `https://github.com/pytorch/pytorch/blob/master/docs/source/distributed.rst#collective-functions`

Reply:
Great suggestion. Added now.